### PR TITLE
[REVIEWS-112] fix time ago issues

### PR DESCRIPTION
**Pull request by Arturo777**
#### What problem is this solving?

Fix the logic to correctly order the words in the translations about the sentence of time ago for each language in the app.

https://arturoreviews--sandboxusdev.myvtex.com/invicta-pro-diver-swiss-movement-quartz-watch---stainless-steel-case---model-0003/p

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -36,9 +36,10 @@
     "vtex.product-review-interfaces": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.product-review-interfaces@1.0.2/public/_types/react",
     "vtex.product-summary": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.product-summary@2.80.1/public/@types/vtex.product-summary",
     "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.132.4/public/@types/vtex.render-runtime",
-    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.49.0/public/@types/vtex.search-graphql",
-    "vtex.store": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.store@2.125.0/public/@types/vtex.store",
-    "vtex.store-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.store-graphql@2.152.2/public/@types/vtex.store-graphql",
+    "vtex.reviews-and-ratings": "https://arturoreviews--sandboxusdev.myvtex.com/_v/private/typings/linked/v1/vtex.reviews-and-ratings@3.10.3+build1660232187/public/@types/vtex.reviews-and-ratings",
```
Comment (btalma) [inline react/package.json:39]: Is it an issue to have the workspace here?

Comment (polishq) [reply react/package.json:39]: Yes, good catch!

Comment (Arturo777) [reply react/package.json]: updated!

File: messages/ko-KR.json [hunk 2]
```diff
@@ -76,20 +76,12 @@
...
-  "store/reviews.list.timeAgo.day": "일",
-  "store/reviews.list.timeAgo.hours": "시간",
-  "store/reviews.list.timeAgo.hour": "시간",
-  "store/reviews.list.timeAgo.minutes": "분",
-  "store/reviews.list.timeAgo.minute": "분",
+  "store/reviews.list.timeAgo.years": "{timeUnits} 년 전",
+  "store/reviews.list.timeAgo.months": "{timeUnits} 개월 전",
+  "store/reviews.list.timeAgo.days": "{timeUnits} 일 전",
+  "store/reviews.list.timeAgo.hours": "{timeUnits} 시간 전",
+  "store/reviews.list.timeAgo.minutes": "{timeUnits} 분 전",
```
Comment (Arturo777) [inline messages/ko-KR.json:84]:
```suggestion
  "store/reviews.list.timeAgo.years": "{timeUnits}년 전",
  "store/reviews.list.timeAgo.months": "{timeUnits}개월 전",
  "store/reviews.list.timeAgo.days": "{timeUnits}일 전",
  "store/reviews.list.timeAgo.hours": "{timeUnits}시간 전",
  "store/reviews.list.timeAgo.minutes": "{timeUnits}분 전",
```

File: messages/bg-BG.json [hunk 3]
```diff
@@ -76,20 +76,14 @@
...
-  "store/reviews.list.timeAgo.hour": "час",
-  "store/reviews.list.timeAgo.minutes": "минути",
-  "store/reviews.list.timeAgo.minute": "минута",
+
+  "store/reviews.list.timeAgo.years": "{timeUnits} {timeUnits, plural, =1 {година} other {години}} преди",
+  "store/reviews.list.timeAgo.months": "{timeUnits} {timeUnits, plural, =1 {месец} other {месеца}} преди",
+  "store/reviews.list.timeAgo.days": "{timeUnits} {timeUnits, plural, =1 {ден} other {дни}} преди",
+  "store/reviews.list.timeAgo.hours": "{timeUnits} {timeUnits, plural, =1 {час} other {часове}} преди",
+  "store/reviews.list.timeAgo.minutes": "{timeUnits} {timeUnits, plural, =1 {минута} other {минути}} преди",
+
```
Comment (Arturo777) [inline messages/bg-BG.json:86]:
```suggestion

  "store/reviews.list.timeAgo.years": "дни {timeUnits} {timeUnits, plural, =1 {година} other {години}}",
  "store/reviews.list.timeAgo.months": "дни {timeUnits} {timeUnits, plural, =1 {месец} other {месеца}}",
  "store/reviews.list.timeAgo.days": "дни {timeUnits} {timeUnits, plural, =1 {ден} other {дни}}",
  "store/reviews.list.timeAgo.hours": "дни {timeUnits} {timeUnits, plural, =1 {час} other {часове}}",
  "store/reviews.list.timeAgo.minutes": "дни {timeUnits} {timeUnits, plural, =1 {минута} other {минути}}",

```

**Review by camilavcoutinho — APPROVED** [review]

**Review by btalma — APPROVED** [review]
lgtm

**Pull request merged**